### ISSUE TITLE
Plans to support CiviCRM 4.4?

### ISSUE THREAD

**Issue by jtbayly** [sciencegalleryderek/com.sciencegallery.mailchimp#18]
I'm super excited about this, but I need to move to Civi 4.4. Does anybody know if it works on 4.4, even though it doesn't say it supports it? I'm happy to test it, but if somebody can just say, "don't bother" that would save me some time.

Also, if it doesn't work, are there plans to get it working soon?

Thanks,
-Joseph

**Comment by sciencegalleryderek** [comment]
HI Joseph,

I'm just in the process of moving our own site to 4.4 so hopefully I'll
have that tested next week

D

On 8 November 2013 16:23, Joseph Bayly notifications@github.com wrote:

> I'm super excited about this, but I need to move to Civi 4.4. Does anybody
> know if it works on 4.4, even though it doesn't say it supports it? I'm
> happy to test it, but if somebody can just say, "don't bother" that would
> save me some time.
> 
> Also, if it doesn't work, are there plans to get it working soon?
> 
> Thanks,
> -Joseph
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/sciencegalleryderek/com.sciencegallery.mailchimp/issues/18
> .

## 
- DEREK WILLIAMS - TECHNICAL MANAGER*
  *SCIENCE GALLERY http://www.sciencegallery.com *| T: + 353 1 896 4198 |
  M: + 353 86 661 3617

## 

_25.10.13 - 19.01.14_*

*
_GROW YOUR OWN..._
_LIFE AFTER NATURE_
*
*
*
What is Science Gallery? http://sciencegallery.com/this_is_science_gallery

*
*

*
**
**
_P__art of the Science Gallery Network pioneered by Trinity College Dublinhttp://www.tcd.ie/. 
G_*enerously supported by Wellcome Trust alongside our Science Circle 
members Deloitte, Google, ICON, NTR Foundation and Pfizer, in addition to 
government support. Our media partner is The Irish Times. Please note that 
email sent to, from or within Science Gallery may be the subject of a 
request under the Freedom of Information Act. *
*
*

**Comment by jtbayly** [comment]
Sweet. I just downloaded it and tested it, and it actually looked like it was pretty close to working. I got a proper warning when I saved the settings to sync a list that didn't exist in Mailchimp. Then I created the list in Mailchimp and tried again, but I got a 403 error when I tried save the settings. Anyway, I don't know if that info is helpful to you, but I thought I'd pass it along.

I'm looking forward to trying it out in 4.4. Let me know if I can be of any help in testing or something.

-Joseph

**Comment by shawnholt** [comment]
Hi Joe @jtbayly - were you able to get this resolved?  Any production use of mailchimp sync w/ 4.4?

**Comment by jtbayly** [comment]
No, I just had enough time to test quickly. I can't work on fixing it now.

**Comment by sciencegalleryderek** [comment]
Hi All,

Just got around to upgrading our own site to 4.4 and this appears to be working ok?  @jtbayly - are you still seeing your issues? What CMS are you using?

**Comment by jtbayly** [comment]
I'm using Drupal 7.23 and CiviCRM 4.4 beta 4. What are you running @sciencegalleryderek ? Maybe I just need to run the latest CiviCRM release...

**Comment by sciencegalleryderek** [comment]
We're on Drupal 7.24 and CiviCRM 4.4.0

**Comment by jtbayly** [comment]
I've got some more info for you, @sciencegalleryderek 

I just updated to Drupal 7.24 and CiviCRM 4.4.3 and I'm seeing the same error when i try to save the settings at civicrm/admin/setting/mailchimp. 

Here is the exact error:
"Sorry but we are not able to provide this at the moment.
We couldn't verify the URL is working. Please double check and try again. HTTP Code: 403
Return to home page."

If I save the page with just an API key, and no lists selected, I don't get an error. But as soon as I select a list, it gives me that error. I'm wondering if I've just missed some step in the setup process. 

Update: I just tried again using Drupal 7.23 and CiviCRM 4.3.7 and I'm seeing the exact same behavior. Is it possible this is communicating on a port I have blocked or something?

Thanks,
-Joseph

**Comment by shawnholt** [comment]
http://en.wikipedia.org/wiki/HTTP_403
looks like a security issue or API / URL setting for credentials ...

**Comment by jtbayly** [comment]
Ok, I found the solution in one of the other issues here on Github. I needed to give anonymous users the "CiviContribute: make online contributions" permission. That fixed it. You probably want to add that to the setup instructions in the readme. We never enable CiviContribute, and that might by why the permission wasn't automatically set (if you are expecting that). 

Thanks,

**Comment by sciencegalleryderek** [comment]
Hi Joseph,

Thanks for this - I'll update the documentation

Derek

On 10 December 2013 18:40, Joseph Bayly notifications@github.com wrote:

> Ok, I found the solution in one of the other issues here on Github. I
> needed to give anonymous users the "CiviContribute: make online
> contributions" permission. That fixed it. You probably want to add that to
> the setup instructions in the readme. We never enable CiviContribute, and
> that might by why the permission wasn't automatically set (if you are
> expecting that).
> 
> Thanks,
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/sciencegalleryderek/com.sciencegallery.mailchimp/issues/18#issuecomment-30255078
> .

## 

_25.10.13 - 19.01.14_

_GROW YOUR OWN..._
_LIFE AFTER NATURE_

*What is Science Gallery? 
http://sciencegallery.com/this_is_science_gallery            *

_P__art of the Science Gallery Network pioneered by Trinity College Dublin 
http://www.tcd.ie/. G_*enerously supported by Wellcome Trust alongside 
our Science Circle members Deloitte, Google, ICON, NTR Foundation and 
Pfizer, in addition to government support. Our media partner is The Irish 
Times. Please note that email sent to, from or within Science Gallery may 
be the subject of a request under the Freedom of Information Act. *

**Comment by tapashdatta** [comment]
thanks for the head up, but does not really work. My civi 4.4.4 anonymous user has permission to access "CiviContribute: make online contributions" but still getting error "We couldn't connect to the specified the URL. Please double check and try again."

**Comment by jtbayly** [comment]
Hmmm... I'm on 4.4.4 too, and I can confirm that it still works exactly the same way. Which version of this extension are you using? I'm still using 1.2...

**Comment by vishram74** [comment]
Hi all, have the issues with Civicrm 4.4 been resolved

**Comment by tapashdatta** [comment]
Personally, I still did not managed to get it work (master or v1.2) with a fresh install of civicdm 4.4.4 . tried many different settings but no avail...
